projects/ngx-location-viewer/src/lib/components/layer-management/layer-management.component.html

```diff
@@ -1,11 +1,30 @@
-<h1 class="has-base-font">Lagen</h1>
-<div class="u-margin-left-xs">
-    <div *ngIf="operationalLayer" class="u-margin-top">
-        <h2 class="has-base-font">Operationele laag:</h2>
-        <aui-layer *ngIf="operationalLayer" [layer]="operationalLayer" [layerType]="layerTypes.operational" (layerVisibiltyChange)="onLayerVisibilityChange($event)"></aui-layer>
+<div class="sidebar">
+    <div class="sidebar__header">
+        <button class="a-button-transparent a-button--default has-icon sidebar-close" aria-label="Close" (click)="toggleLayermanagement()">
```

Comment:
I can't see it in your code, but is this button aligned to the right? If not, add the `.u-text-right` class.

Reply:
positioning of button is fixed in css, to counter padding of parent div